pkg/gokeyless.sysv

```diff
@@ -61,28 +53,14 @@ start() {
     return 1;
   fi
 
-  if [ ! -f "$SERVER_KEY" ]; then
-    if [ ! -d `dirname $SERVER_KEY` ]; then
-      mkdir `dirname $SERVER_KEY` || return 1
-    fi
-
-    if [ ! -d `dirname $SERVER_CERT` ]; then
-      mkdir `$dirname SERVER_CERT` || return 1
-    fi
-
-    $program -init -key="${SERVER_KEY}" -cert="${SERVER_CERT}"
-
-  fi
-
   [ -f $pidfile ] && rm -f $pidfile
 
+  cd $WORKDIR
```

Comment:
We could check here if `token.json` exists, and then create it if it doesn't. I mainly didn't want to marshal JSON from Bash.